src/NodaTime.Test/TimeZones/BclDateTimeZoneTest.cs

```diff
@@ -134,13 +146,13 @@ public void DateTimeMinValueStartRuleExtendsToBeginningOfTime()
         [Test]
         public void Equality()
         {
-            if (BclZonesOrEmptyOnMono.Count < 2)
+            if (BclZonesOrJustNullOnMono.Count < 2)
```

Comment:
Why not? Basically if we don't have two time zones to compare, we don't compare them. Fortunately, a single null reference still satisfies the "we don't have enough to compare" constraint in the same way that an empty collection does.